models/fixtures/user.yml

```
@@ -187,17 +187,17 @@
 
 -
   id: 6
-  lower_name: user6
-  name: user6
+  lower_name: org6
+  name: org6
   full_name: User Six
```

Shoudn't we change this too for consistency.
```suggestion
  full_name: Org Six
```
I like these changes, we need more unit test cases in Gitea. Appreciated for your initiate.
I will take a closer look today. Could you please wait for my review. If it is on priority you can go ahead and merge after other maintainers approvals.

---

Furture related :-  Could you slice this PR into multiple Pr. easy for me to keep track of changes, Thare are 90+ files. So.

---

Thanks for your patient review. As I mentioned in the PR information, there are only three non-trivial changes that are not 'find and replace' in IDE. I don't think this can break into different parts because it's weird that both 'user3' and 'org3' are referred to the same user.

---

> Shoudn't we change this too for consistency.
> 
> I like these changes, we need more unit test cases in Gitea. Appreciated for your initiate. I will take a closer look today. Could you please wait for my review. If it is on priority you can go ahead and merge after other maintainers approvals.

Not in a rush. Take your time :+1:

---

>  I don't think this can break into different parts because it's weird that both 'user3' and 'org3' are referred to the same user.
You can use git blame to root this.